x/bep3/keeper/swap.go

```diff
@@ -64,11 +64,15 @@ func (k Keeper) CreateAtomicSwap(ctx sdk.Context, randomNumberHash []byte, times
 		}
 		err = k.IncrementIncomingAssetSupply(ctx, amount[0])
 	case types.Outgoing:
+		// Amount in outgoing swaps must be greater than the deputy's fixed fee. Incoming swaps
+		// have already had their fees collected by the deputy during the relay process.
```

Comment:
nit: move comment about Incoming swaps to that case

Reply:
updated in latest